mlflow/gateway/providers/base.py

```diff
@@ -0,0 +1,26 @@
+from abc import ABC
+from typing import Tuple
+
+from ..schemas import chat, completions, embeddings
+from ..config import RouteConfig
+
+
+class BaseProvider(ABC):
+    """
+    Base class for MLflow Gateway providers.
+    """
+
+    NAME: str
+    SUPPORTED_TASKS: Tuple[str, ...]
```

Comment:
```suggestion
    SUPPORTED_ROUTE_TYPES: Tuple[str, ...]
```

Is `SUPPORTED_ROUTE_TYPES` more accurate?

Reply:
I like the sound of it

Reply:
Renamed